etp-backend/src/main/clj/solita/etp/service/laskutus.clj

```diff
@@ -377,9 +391,8 @@
                                        laskutustieto-xml-files
                                        laskutustieto-destination-dir)
               (log/info "Laskutustieto xmls uploaded with SFTP."))
-
-            ;; TODO send täsmätytysraportti with SMTP.
-            )
+            (send-tasmaytysraportti-email! tasmaytysraportti-file)
+            (log/info "Täsmätysraportti sent as an email"))
```

Comment:
It's just a nice to have, but might be nice to one or both of the destination address and the message ID that SMTP servers give us when they accept messages in.